docs/src/main/sphinx/admin/properties-query-management.md

```diff
@@ -97,11 +97,11 @@ Configures the behavior to handle killing running tasks in the event of low
 memory availability. Supports the following values:
 
 - `none` - Do not kill any tasks in the event of low memory.
-- `total-reservation-on-blocked-nodes` - Kill the tasks which are part of the queries
-  which has task retries enabled and are currently using the most memory specifically
+- `total-reservation-on-blocked-nodes` - Kill the tasks that are part of the queries
+  which has task retries enabled and is currently using the most memory specifically
```

Comment:
leave plural everywhere

 Kill the tasks that are part of the queries
  which have task retries enabled and are currently using the most memory, specifically